readthedocs/projects/models.py

```diff
@@ -1055,10 +1055,24 @@ def get_git_service_class(self, fallback_to_clone_url=False):
         return service_cls
 
     @property
-    def git_provider_name(self):
```

Comment:
Have you grepped all the repositories to check if we are using it in other places? Just in case.

Reply:
nothing in .com and ext.